portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,33 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
typically you do not want the TODO in the java doc. I would do:

/** Servlet ... */
// TODO: Add support for handling comment data.

Reply:
Got it!
Done